vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml

```diff
@@ -180,7 +180,7 @@ jobs:
           IS_BASIC_TIER_HUB: $(IS-BASIC-TIER-HUB)
           IOTHUB_CLIENT_SECRET: $(IOTHUB-CLIENT-SECRET)
           IOTHUB_CLIENT_ID: $(IOTHUB-CLIENT-ID)
-          IOTHUB_TENANT_ID: $(IOTHUB-TENANT-ID)
```

Comment:
Can you check if java uses this anywhere? LA_AAD_TENANT
If yes, I'm changing the script to use the same MSFT_TENANT_ID. So you might want to change that.

Reply:
Nope, no references found for "LA_AAD_TENANT" in this repo